docs/exceptions.md

```diff
@@ -71,6 +72,43 @@ ValueError: 2 + 2 != 4
 
 ```
 
+## Defining custom error types
+
+To define custom errors, you need to define a function that returns a string as the
+error message.
+
+## Example
+
+```js linenums="1" title="arg-error.rn"
+fun FunctionError(func, base_err) -> "Something went wrong in " + func + ": " + base_err 
+
+fun risky_operation(y)
+{
+    try
+    {
+        x = 10 / y
+    }
+    catch as e
+    {
+        raise FunctionError("risky_operation", e)
+    } 
+}
+
+risky_operation(0)
```

Comment:
Please test your code before putting on public docs.

Also follow K&R formatting for writing code in docs.